esp/src/eclwatch/QuerySetDetailsWidget.js

```diff
@@ -262,6 +263,9 @@ define([
                 this.librariesUsedTab.set("tooltip", tooltip);
             } else if (name === "Clusters") {
                 if (lang.exists("ClusterQueryState.length", newValue)) {
+                    newValue.ClusterQueryState[0].MixedNodeStates ? dom.byId(this.id + "SuspendCluster").src = dojoConfig.getImageURL("mixwarn.png") : dom.byId(this.id + "SuspendCluster").src = dojoConfig.getImageURL("errwarn.png");
```

Comment:
"Clusters" will only come back in two cases. 1. If the query is suspended by cluster or 2. If it is in mixed node state otherwise that will not be in the response therefore all the logic inside which includes any errors in errors tab will be empty.